src/main/drivers/timer.h

```diff
@@ -107,6 +111,26 @@ typedef struct timerHardware_s {
 #endif
 } timerHardware_t;
 
+#ifdef USE_TIMER_MGMT
+
+typedef uint8_t timerTag_t;
+
+typedef struct timerChannelConfig_s {
```

Comment:
This is bad from user perspective - this type should contain only information necessary for unique identification of timer channel configuration:

`AF` + `inverted` can be inferred from timer-channel-pin (and validity should be checked)

DMA needs full triple, but at most 3 variants are possible - maybe just store variant index (autoconfiguration will be even better, but complicated)

Also this expects all possible combinations are allocated (TIMER_CHANNEL_COUNT). If timer channel is included, user may configure only needed channels (but I am not sure if this indirection is desired)